core/templates/components/question-directives/questions-list/questions-list.component.ts

```diff
@@ -425,13 +426,33 @@ angular.module('oppia').component('questionsList', {
         }
         $location.hash(ctrl.questionId);
       };
-      ctrl.deleteQuestionFromSkill = function(
+
+      ctrl.deleteQuestionFromSkill = async function(
```

Comment:
This function s made `async` because in line 439 an async function is called for which we have to wait until its completion, and `await` keyword is only allowed in async methods.